From 158d0bc3a2a7a93090cc12c5b0dd3c27e6f44d61 Mon Sep 17 00:00:00 2001
From: Carlos Santos <unixmania@gmail.com>
Date: Mon, 23 Dec 2019 08:02:19 -0300
Subject: [PATCH] Temporary fix for build without C++

C++ is required only for the fuzzing tests but AC_PROG_CXX is included
by configure.ac even when fuzzing is not enabled (which we don't do on
Buildroot).

The patch applied upstream had issues and was reverted[1]. Use a local
patch to solve the problem temporaryly.

Fixes:
    http://autobuild.buildroot.net/results/13f5e37b47b255da4158bec34e5459136f7e60d4
    http://autobuild.buildroot.net/results/1c26db2509c79e00c0de1165945277eaa57b149f
    http://autobuild.buildroot.net/results/b7b6b7b7aca79e847b442cbd2305427d91fe5d70
    http://autobuild.buildroot.net/results/1cd5a82a0e799aa5027e2e2c03b246332cc3a15d
    http://autobuild.buildroot.net/results/d7ec878907f714377c83e9a496e97cbf9382d787
    http://autobuild.buildroot.net/results/1c7f0c1b3ce4871cd87bd6059b1f0a6dc4e74a9c
    http://autobuild.buildroot.net/results/196b81d580325607c8da90beeb79e1f6b8ab8b47
    http://autobuild.buildroot.net/results/f90f7b4ac710b56686635f8ae27059c11b963e47

1. https://github.com/tpm2-software/tpm2-tss/commit/60c26e4c4faba6ba12469485653e17092b510840

Signed-off-by: Carlos Santos <unixmania@gmail.com>
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index cd50451d..a3bd7144 100644
--- a/configure.ac
+++ b/configure.ac
@@ -25,7 +25,6 @@ AC_CANONICAL_HOST
 AX_IS_RELEASE(dash-version)
 AX_CHECK_ENABLE_DEBUG([info])
 
-AC_PROG_CXX
 AC_PROG_CC
 AC_PROG_LN_S
 AC_USE_SYSTEM_EXTENSIONS
-- 
2.18.1

